### ISSUE TITLE
[Bug]: jest config cannot be used if it's located at the monorepo's root

### ISSUE THREAD

**Issue by LukeChastainKodiak** [aspect-build/rules_jest#86]
### What happened?

In a monorepo, jest_test fails when jest.config.js is located at the root instead of the local directory.

The following is given:
```
Expected to find file jest.config.js in //monrepo/someapp/, but instead it is in //
To use copy_to_bin, either move jest.config.js to //monorepo/someapp, or move the copy_to_bin target to //
```
__

### Version

Development (host) and target OS/architectures:

Output of `bazel --version`: bazel 5.1.1

Version of the Aspect rules, or other relevant rules from your
`WORKSPACE` or `MODULE.bazel` file:
rules_js-1.15.0
rules_jest-0.14.2

Language(s) and/or frameworks involved:

### How to reproduce

Using https://github.com/aspect-build/rules_jest/tree/main/example/snapshots

1. move jest.config.js to the root of the repo
2. Edit BUILD.bazel at the root of the repo to export jest.config.js
3. Change BUILD.bazel in snapshots to `config = //:jest.config.js`

```shell

```

### Fund our work

- [ ] Sponsor our open source work by donating a [bug bounty](https://opencollective.com/aspect-build/)

**Comment by jbedard** [comment]
Since the config file is in a different directory then the BUILD you can't access it as source and it needs to be in the output directory. See the message in the error you pasted.

Here's an example/test: https://github.com/aspect-build/rules_jest/pull/88
